Sources/LiveViewNative/Property Wrappers/LiveContext.swift

```diff
@@ -58,14 +58,13 @@ public struct LiveContext<R: RootRegistry>: DynamicProperty {
         return coordinator.builder.fromNodes(element.children(), context: storage)
     }
     
-    private static func elementWithName(
-        _ tagName: String,
-        namespace: String?
+    private static func isTemplateElement(
+        _ template: String
     ) -> (NodeChildrenSequence.Element) -> Bool {
         { child in
             if case let .element(element) = child.data,
-               element.namespace == namespace,
-               element.tag == tagName
+               element.attributes.first(where: { $0.name == "template" })?.value == template
+                || element.attributes.contains(where: { $0.name == .init(stringLiteral: "#\(template)") })
```

Comment:
I was under the impression that the # syntax was going to be an EEx engine transformation to the same template attribute?

Reply:
I'm not sure how to implement it that way with the `TagEngine`. @supernintendo Is this possible and if so can you help with this?

Reply:
@shadowfacts @carson-katri Yeah, we probably want to handle this within `TagEngine` to make it available to all platforms. The idea is to convert `<Text #label>` to `<Text template={:label}>`, correct?

Reply:
Yeah, that's my understanding

Reply:
@carson-katri @shadowfacts Looking into this further, it is not currently possible to override how attributes are handled within elements from implementations of `Phoenix.LiveView.TagEngine`. `Phoenix.LiveView.TagEngine` _does_ do some parsing on attributes [here](https://github.com/phoenixframework/phoenix_live_view/blob/v0.18.18/lib/phoenix_live_view/tag_engine.ex#L934) but its not exposed through a callback in a way that we could hook into it.

I'm happy to open a PR on [phoenix_live_view](https://github.com/phoenixframework/phoenix_live_view/) to convert some of the attribute parsing functions into callbacks so we can override them, although no guarantees that will get merged. I think its probably fine to handle this within the client for now until we know we can do that. Thoughts @bcardarella?

Reply:
I could remove the shorthand until we know whether or not LiveView will accept the changes, then we can reevaluate then.

Reply:
This is such an elegant solution that really makes the project more approachable for new learners. it would be worth to having it in client and handle technical implementation later as @supernintendo mentioned.